scripts/minishift-all-tests.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+executing() {
+   set +x
+  echo -e "\n------------------------------\n${1}\n------------------------------\n"
+   set -x
+}
+
+set -ex
+
+#Export GitHub token to avoid 
+executing "Setting up environment..."
+
+export PATH="$PATH:/usr/local/go/bin/"
+export GOPATH=$HOME/go
+git clone https://github.com/openshift/odo.git openshift/odo
+cd openshift/odo
+
+mkdir -p $GOPATH/bin
+make goget-ginkgo
+export PATH="$PATH:$(pwd):$GOPATH/bin"
+
+executing "Building ODO..."
+make bin
+sudo cp odo /usr/bin
```

Comment:
why are we copying it in "/usr/bin", shouldn't we use "$GOPATH/bin"?

Reply:
$GOPATH/bin holds go binaries. In general all applications go to /usr/bin. 
The same is done for minikube

Reply:
If we are creating a fresh vm every time we test then it makes sense but if we using a long running cluster we should use  "$GOPATH/bin".
what do you think?

Reply:
agreed, I will make the change

Reply:
change made